lib/src/cli.rs

```diff
@@ -611,7 +612,7 @@ fn prepare_for_write() -> Result<()> {
     if ostree_ext::container_utils::running_in_container() {
         anyhow::bail!("Detected container; this command requires a booted host system.");
     }
-    if !std::path::Path::new("/run/ostree-booted").exists() {
+    if !is_ostree_booted()? {
         anyhow::bail!("This command requires an ostree-booted host system");
     }
```

Comment:
"if condition then bail" can be ensure

```suggestion
    anyhow::ensure!(is_ostree_booted()?, "This command requires an ostree-booted host system");
```

(If we're already changing this code)